support-files/mysql.spec.sh

```diff
@@ -1087,21 +993,19 @@ echo "====="                                     >> $STATUS_HISTORY
 #  Files section
 ##############################################################################
 
-%files -n MySQL-server%{product_suffix} -f release/support-files/plugins.files
+%files -n mysql-wsrep-server%{product_suffix} -f release/support-files/plugins.files
 %defattr(-,root,root,0755)
 
 %if %{defined license_files_server}
 %doc %{license_files_server}
 %endif
-%doc %{src_dir}/Docs/ChangeLog
-%doc %{src_dir}/Docs/INFO_SRC*
+%doc Docs/ChangeLog
+%doc release/Docs/INFO_SRC*
```

Comment:
I'm not happy with this change: "INFO_SRC" is not related to the compilation, it describes the source tarball - that's why the Oracle build procedure includes it in the tarball.
IMO, we should see to achieve the same in the long run, and for now we might create it below "%{src_dir}/" in the "build" or "setup section if it is missing.